languages/README.md

```diff
@@ -0,0 +1,128 @@
+to translate the spec into a specific language since the language barrier might 
+hinder the spread of CloudEvents.
+
+This page contains information about how to translate a document from scratch, 
+definition of roles and responsibilities as well as guidelines that translators 
+and reviewers should follow.
+
+## Roles
+
+There are two categories of roles:
+
+1. **Translator.** This is anyone who participates in the translation of any 
+   CloudEvents documents. One becomes a formal translator once a Pull Request(PR) 
+   of his/her translation is merged.
+
+2. **Reviewer.** A reviewer is responsible for checking new translation, ensuring 
+   translation up to date as English version changed and answering related issues.
+   A translation will not be merged until reviewers give their LGTM.
+   Check current reviewers [here](reviewers.md).
+
+### Translators
+
+A Translator can start translate following [how to translate](#how-to-translate).
+
+A translator should follow [translation style guide](#translation-style-guide).
+
+### Reviewers
+
+The following rules govern adding and removing reviewers:
+
+- New reviewers can be proposed via a PR to edit the [reviewers](reviewers.md) file.
+  Only probationary reviewers can be assigned as a formal reviewer. 
+  A translator who completes at least 6 translation documents becomes a probationary 
+  reviewer automatically.
+- Reviewers can be removed via a PR to edit the [reviewers](reviewers.md) file. 
+  That happens when a reviewer explicitly give up the role or has not performed 
+  reviewer duties over 2 weeks.
+
+## How to translate
+
+### Step 1. Check the Languages.md
+First, fork [cloudevents/spec](https://github.com/cloudevents/spec/tree/main) 
+to your Github repo. Then check the[languages.md](languages.md)to see if the language 
+you'd like to translate into already exists. If it exists, add new translation under 
+`languages/$(your-language-code)/$(corresponding-position)`. 
+For example:
+```
+cloudevents/spec.md should be translated as languages/zh-cn/cloudevents/spec.md
+cloudevents/adapters/github.md should be translated as languages/zh-cn/cloudevents/adapters/github.md
+*****NOTE: documents under languages directory should not be translated ***** 
+```
+If your language doesn't exist, add a new item to the [languages.md](languages.md) file, 
+which links to a newly-created directory named after the corresponding language code within 
+[language codes](https://en.wikipedia.org/wiki/Language_localisation#Language_tags_and_codes). 
+Then, add new translation to right position as described above.
+
+### Step 2. choose which document you'd like to translate
+A translator can know about which documents are not translated yet by reading a 
+"CE translation list".
+
+The list looks like this:
+
+![CloudEvents translation list](CE_translation_list.png)
+
+- version: choose from {`finished`, `working draft`}; a translator should always 
```

Comment:
Even "finished" documents in English can change though. I think we'll want some sort of tracking for "the version that was last translated".

Reply:
I added last edit time in `languages/{language-code}/translations.md` to see when that document was last translated.
Initially, the last edit time will be the time when the translation PR is merged.
Reviewers should ensure translations are up-to-date as original one changes. And then they update the last edit time.